.github/workflows/release.yml

```diff
@@ -101,7 +101,6 @@ jobs:
     steps:
       - uses: actions/download-artifact@v4
         with:
-          merge-multiple: true
```

Comment:
Can you also update the anaconda command to 
```
anaconda --token ${{ secrets.ANACONDATOKEN }} upload --user scipp --label main $(find . -type f -name "*.tar.bz2")
```
This should get all the files out with the extension we are looking for.
Or we can come up with the right path looking at https://github.com/scipp/scipp/blob/9603035e0b57b837c594c05f4461849cec3202d0/.github/workflows/conda.yml#L53